test/thundercat-router-test.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
```

Comment:
Tests: -1
I'm sure you know this but, you're missing your additional 400/404/204 tests. I can help you out with those if you need it.